src/libraries/System.Text.Json/gen/JsonSourceGenerator.Emitter.cs

```diff
@@ -152,7 +152,8 @@ private void AddSource(string fileName, string source, bool isRootContextDef = f
                 bool isInGlobalNamespace = @namespace == JsonConstants.GlobalNamespaceValue;
 
                 StringBuilder sb = new(@"// <auto-generated/>
-#nullable enable
+#nullable enable annotations
```

Comment:
If we only enable annotations, isn't disabling warnings a redundant declaration?

Reply:
since this is generating source code and we don't know what is the default for specific project I believe we need to be explicit here

Reply:
Ah, I see what you mean.